packages/react-query/src/server/ssgProxy.ts

```diff
@@ -42,7 +42,7 @@ import {
 interface CreateSSGHelpersInternal<TRouter extends AnyRouter> {
   router: TRouter;
   ctx: inferRouterContext<TRouter>;
-  transformer?: ClientDataTransformerOptions;
+  transformer?: DataTransformerOptions;
```

Comment:
I'm not completely sure of the differences between all the different DataTransformer types to be fair, so this should probably be validated @KATT